core/node/node_stats_tracker.go

```diff
@@ -147,3 +198,33 @@ func (m *StatsTracker) ConsumersCount(rangeTime string) (ConsumersCount, error)
 
 	return ConsumersCount{}, errors.New("identity not found")
 }
+
+// SeriesEarnings retrieves and resolved earnings data series metrics during a time range
+func (m *StatsTracker) SeriesEarnings(rangeTime string) (SeriesEarnings, error) {
+	id, ok := m.currentIdentity.GetUnlockedIdentity()
+	if ok {
+		return m.providerSeriesEarnings(id, rangeTime)
+	}
+
+	return SeriesEarnings{}, errors.New("identity not found")
```

Comment:
This error is repeated many times, probably it should be extracted as a constant.